src/main/java/com/cloudant/client/org/lightcouch/CouchDbClient.java

```diff
@@ -89,11 +56,8 @@
  * .replicator()}
  * <p/>
  * <p/>
- * <p>At the end of a client usage; it's useful to call: {@link #shutdown()} to ensure proper
- * release of resources.
  *
  * @author Ahmed Yehia
- * @see CouchDbClientAndroid
  * @since 0.0.2
  */
 public class CouchDbClient extends CouchDbClientBase {
```

Comment:
Yeah, I believe the structure was such that the Android and Java sub-classes of `CouchDbClientBase` could each provide some specific behaviour whilst sharing a bunch of code from the abstract base class. Hopefully we can keep all the Android specifics in the HTTP classes so we don't need the division.